patches/chromium/printing.patch

```diff
@@ -932,3 +932,19 @@ index 63f170c95050416c595e62f4c460c4cd6b7dbd1c..157e3d046889f9c63fdf0fd5d503890f
    // Does bookkeeping when an error occurs.
    virtual mojom::ResultCode OnError();
  
+diff --git a/printing/printing_context_linux.cc b/printing/printing_context_linux.cc
+index fa55d2a1c5f4207cb8eda9305e8bf2af29352c91..13ca65c2c7c3c24882061d8e224bf93899d9a7ab 100644
+--- a/printing/printing_context_linux.cc
++++ b/printing/printing_context_linux.cc
+@@ -52,9 +52,8 @@ void PrintingContextLinux::AskUserForSettings(int max_pages,
+                                               bool is_scripted,
+                                               PrintSettingsCallback callback) {
+   if (!print_dialog_) {
+-    // Can only get here if the renderer is sending bad messages.
+-    // http://crbug.com/341777
+-    NOTREACHED();
```

Comment:
Can we have a custom state that gets passed from the callsite initiating print to distinguish a valid call from us vs the unexpected bad renderer call that upstream expects. I am concerned having both code reach the same block.

Reply:
@deepak1556 we could try, but we patch enough printing code that this isn't really a new surface for error and would add a ton of printing patch for little to no benefit (since initializing the dialog is the only real error we'd see here). What specifically are you concerned about?

Reply:
Upstream usually terminates the services (utility process) when they think the renderer shouldn't gain access to privileges it isn't meant to. Based on the code comment this is deemed one such path. My concern is we are mixing a valid path with an invalid path determined by upstream. Are we certain a bad renderer cannot trigger this path in our case ?

Reply:
Yes, in that with the OOP case the path they consider a bad path is what we specifically want and need, unfortunately.